packages/react-api/src/util/keyringFormat.ts

```diff
@@ -0,0 +1,14 @@
+import { KeyringPair$Json } from '@polkadot/keyring/types';
+
+export function updateToV3KeyringFormat(json: KeyringPair$Json) {
```

Comment:
When you have two different types then you can do:
export function updateToV3KeyringFormat(json: KeyringPair$Json | KeyringPairJson2): json is KeyringPairJson2 {
....
}

Reply:
done